aws-rds-dbclusterparametergroup/src/main/java/software/amazon/rds/dbclusterparametergroup/BaseHandlerStd.java

```diff
@@ -150,17 +160,13 @@ protected ProgressEvent<ResourceModel, CallbackContext> applyParameters(final Am
                                                                             final ProxyClient<RdsClient> proxyClient,
                                                                             final ResourceModel model,
                                                                             final CallbackContext callbackContext) {
-        //isParametersApplied flag for unit testing
-        if (callbackContext.isParametersApplied())
-            return ProgressEvent.defaultInProgressHandler(callbackContext, NO_CALLBACK_DELAY, model);
-        callbackContext.setParametersApplied(true);
```

Comment:
This is the most essential part of this PR.

Reply:
DBParameterGroup does not have a stabilization step, but features the same piece of code.

In the future PR we should replicate the logic to DBParameterGroup

Reply:
Non-blocking for this PR, resolving.